src/containers/Dashboard/style.scss

```diff
@@ -27,4 +27,9 @@
     font-size: 10px;
     padding: 0px;
   }
+}
```

Comment:
Might be better to create a new class for these changes rather than changing `antd`'s builtin classes ?